src/utils/wallet.js

```diff
@@ -157,6 +157,13 @@ class Wallet {
         await (await this.getAccount(this.accountId)).sendMoney(receiverId, amount)
     }
 
+    async transferAllFromLockup() {
```

Comment:
@Patrick1904 nit: I think it's best to just inline it in Redux action, no need to add more stuff to wallet object